src/js/components/IRSReport.jsx

```diff
@@ -42,47 +44,47 @@ const IRSReport = (props) => {
               return <tr key={i}>
                 <td>{msa.id}</td>
                 <td>{msa.name}</td>
-                <td>{msa.totalLars}</td>
-                <td>{msa.totalAmount}</td>
-                <td>{msa.conv}</td>
-                <td>{msa.FHA}</td>
-                <td>{msa.VA}</td>
-                <td>{msa.FSA}</td>
-                <td>{msa.oneToFourFamily}</td>
-                <td>{msa.MFD}</td>
-                <td>{msa.multiFamily}</td>
-                <td>{msa.homePurchase}</td>
-                <td>{msa.homeImprovement}</td>
-                <td>{msa.refinance}</td>
+                <td>{msa.totalLars.toLocaleString()}</td>
+                <td>{msa.totalAmount.toLocaleString()}</td>
```

Comment:
I think the code here could be much more terse (eg, the sorting specified in an array then the msa and totals objects could be parsed into their own sorted arrays and mapped into `td`s